doc/api/cli.md

```diff
@@ -427,6 +427,13 @@ added: v6.0.0
 
 Silence all process warnings (including deprecations).
 
+### `--force-context-aware`
+<!-- YAML
+added: REPLACEME
+-->
+
+Disable loading non-context-aware native addons.
```

Comment:
Maybe add a few words describing what a non-context-aware native addon is and why you might want to stop them from being loaded?

(I realize the second part might be editorializing. Feel free to leave it out if you think it's a bad idea.)